ui_framework/components/global_styles/variables/_z_index.scss

```diff
@@ -0,0 +1,5 @@
+$kuiZContent:         1;
```

Comment:
Technically I think this should be `0`, to create even increments between levels.